src/nccl_ofi_topo.c

```diff
@@ -18,8 +18,13 @@
 #include "nccl_ofi_topo.h"
 #include "nccl_ofi_math.h"
 
+#if HAVE_CUDA
 static const uint8_t display_controller_class_id = 0x03;
 static const unsigned short nvidia_vendor_id = 0x10de;
+#else
+static const uint8_t system_peripheral_class_id = 0x08;
```

Comment:
I wonder if it wouldn't be better to just have these named `target_class_id` and `target_vendor_id` and then not having the compile time conditional in `is_accelerator_dev()`.

Reply:
Yes, that is better, will fix.